com.fyralabs.Exponential.metainfo.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.fyralabs.Exponential</id>
+  <name>Exponential</name>
+  <developer id="com.fyralabs">
+    <name>Fyra Labs</name>
+  </developer>
+  <summary>A purpose-built tool for planning and building products</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+  <url type="homepage">https://linear.app</url>
```

Comment:
why doesn't this correspond to the app-id?

Reply:
Good point, might be best to just omit it. The desktop app wraps the Linear app, which is why I put it here.